src/components/Button/Addon/test.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import Addon from './';
+// import style from './style.css';
```

Comment:
needed?

Reply:
Nah, it's a placeholder in hopes we get css module imports working in tests. You think I should remove it?